voltdb/metadata.csv

```diff
@@ -56,5 +56,49 @@ voltdb.table.tuple_allocated_memory,gauge,,kibibyte,,Total size of memory alloca
+voltdb.idletime.stddev,gauge,,microsecond,,"The standard deviation of the waiting time (in microseconds).",0,voltdb,idletime stddev
+voltdb.idletime.wait.count,count,,,,"The number of times the execution site had to wait for a new task (that is, the queue was empty).",0,voltdb,idletime count
+voltdb.idletime.wait.pct,gauge,,,,"The percentage of time the execution site was waiting for a new task (that is, the site was "idle").",0,voltdb,idletime percent
+voltdb.idletime.wait.total,gauge,,,,"The cumulative number of times the execution site had to wait for a new task (that is, the queue was empty).",0,voltdb,idletime count
+voltdb.import.failures.count,count,,,,"The number of import transactions that failed.",0,voltdb,import failures
+voltdb.import.failures.total,gauge,,,,"The cumulative number of import transactions that failed.",0,voltdb,import failures
+voltdb.import.outstanding_requests,gauge,,record,,"The number of records read from the import stream and waiting to be inserted into the database.",0,voltdb,import outstanding requests
+voltdb.import.retries.count,count,,,,"The number of attempts to replay failed transactions.",0,voltdb,import retries
+voltdb.import.retries.total,gauge,,,,"The cumulative number of attempts to replay failed transactions.",0,voltdb,import retries
+voltdb.import.successes.count,count,,,,"The number of import transactions that succeeded.",0,voltdb,import success
+voltdb.import.successes.total,gauge,,,,"The cumulative number of import transactions that succeeded.",0,voltdb,import success
+voltdb.procedureoutput.avg_result_size,gauge,,byte,,"The average result set size in bytes.",0,voltdb,procedureoutput avg result
+voltdb.procedureoutput.invocations.count,count,,,,"The number of invocations of this procedure.",0,voltdb,procedureoutput invocation
+voltdb.procedureoutput.invocations.total,gauge,,,,"The cumulative number of invocations of this procedure.",0,voltdb,procedureoutput avg invocation
+voltdb.procedureoutput.max_result_size,gauge,,byte,,"The maximum result set size in bytes.",0,voltdb,procedureoutput max result
+voltdb.procedureoutput.min_result_size,gauge,,byte,,"The minimum result set size in bytes.",0,voltdb,procedureoutput min result
+voltdb.procedureoutput.total_result_size,gauge,,mebibyte,,"The total output returned by all invocations of this stored procedure measured in megabytes.",0,voltdb,procedureoutput total result
+voltdb.procedureoutput.weighted_perc,gauge,,,,"A weighted average expressed as a percentage of the result set size returned by invocations of this stored procedure compared to all stored procedure invocations.",0,voltdb,procedureoutput weighted result
+voltdb.procedureprofile.aborts.count,count,,,,"The number of times the procedure was aborted.",0,voltdb,procedureprofile aborts count
+voltdb.procedureprofile.aborts.total,gauge,,,,"The number of times the procedure was aborted.",0,voltdb,procedureprofile aborts count
```

Comment:
Can we express the difference between the count and the total/gauge metrics in the descriptions? (the two descriptions are currently the same)